test/CMakeLists.txt

```
@@ -113,22 +96,24 @@ if(ENABLE_RISCV_TESTS)
         "${DOWNLOAD_URL}"
     )
 
-    file(GLOB elfs_rv32d CONFIGURE_DEPENDS LIST_DIRECTORIES false
+    file(GLOB elfs_rv32 CONFIGURE_DEPENDS LIST_DIRECTORIES false
         "${DOWNLOAD_PATH}/${TARBALL_NAME}/rv32*"
     )
 
-    file(GLOB elfs_rv64d CONFIGURE_DEPENDS LIST_DIRECTORIES false
+    file(GLOB elfs_rv64 CONFIGURE_DEPENDS LIST_DIRECTORIES false
         "${DOWNLOAD_PATH}/${TARBALL_NAME}/rv64*"
     )
 
-    foreach(arch IN ITEMS "rv32d" "rv64d")
-        foreach(elf IN LISTS elfs_${arch})
+    foreach(xlen IN ITEMS 32 64)
```

You could pull these two globs into the `foreach` loop like you did above:
```suggestion
    foreach(xlen IN ITEMS 32 64)
      file(GLOB elfs_rv${xlen} CONFIGURE_DEPENDS LIST_DIRECTORIES false
  "${DOWNLOAD_PATH}/${TARBALL_NAME}/rv${xlen}*"
  )
```